src/tests/modules/bson.rs

```diff
@@ -306,3 +307,18 @@ fn system_time() {
         0
     );
 }
+
+#[test]
+fn debug_print() {
+    #[derive(Debug)]
+    struct Example {
+        a: i32,
+        b: String,
+    }
+
+    let doc = doc! {"a": 1, "b": "data"};
```

Comment:
tiny nitpick: we usually add a space after the opening bracket and a space before the closing bracket when invoking this macro, so this should be `doc! { "a": 1, "b": "data" };`